pay-api/src/pay_api/services/payment_transaction.py

```diff
@@ -353,11 +353,12 @@ def update_transaction(transaction_id: uuid,  # pylint: disable=too-many-locals
         if payment.payment_status_code == PaymentStatus.COMPLETED.value:
             # if the transaction status is EVENT_FAILED then publish to queue and return, else raise error
             if transaction_dao.status_code == TransactionStatus.EVENT_FAILED.value:
+                transaction_dao.status_code = TransactionStatus.COMPLETED.value
```

Comment:
This order change was causing an error in prod, where the queue was down and next time when stale payment job runs, wasn't publishing the message.